src/LayerEditor/ui/data/regions_model.py

```diff
@@ -15,25 +15,26 @@ def __init__(self, le_data, regions_data):
 
         for data in regions_data:
             self.add_region_from_data(data)
-        Region.none = self.regions.get(0)
+        RegionItem.none = self.regions.get(0)
 
     def add_region_from_data(self, region_data):
```

Comment:
This is not and 'adder' should not be method of this class.

Reply:
Actually it was adder, but it has been changed since then. Also this method is copying the object and therfore it was renamed to `copy_region_from_data()`